frontend/src/container/FormAlertRules/ChQuerySection.tsx

```diff
@@ -0,0 +1,53 @@
+import ClickHouseQueryBuilder from 'container/NewWidget/LeftContainer/QuerySection/QueryBuilder/clickHouse/query';
+import { IClickHouseQueryHandleChange } from 'container/NewWidget/LeftContainer/QuerySection/QueryBuilder/clickHouse/types';
+import React from 'react';
+import { IChQueries } from 'types/api/alerts/compositeQuery';
+
+function ChQuerySection({
+	chQueries,
+	setChQueries,
+}: ChQuerySectionProps): JSX.Element {
+	const handleChQueryChange = ({
+		rawQuery,
+		legend,
+		toggleDelete,
+	}: IClickHouseQueryHandleChange): void => {
+		let chQuery = chQueries.A;
```

Comment:
can we make a util for this one ?

Reply:
Not sure what you mean. please explain... Do you mean have a util to convert IClickhouseQuery to IChQuery? I think the operation is pretty local to include in utils.